File: hospital-management-api/Models/Appointment.cs
using System;

namespace hospital_management_api.Models
{
    public class Appointment
    {
        public int Id { get; set; }
        public DateTime CreatedOn { get; set; }
        public string CreatedBy { get; set; }
        public int PatientId { get; set; }
        public int DoctorId { get; set; }
        public int HospitalId { get; set; }
        public DateTime AppointmentTime { get; set; }
        public string AppointmentType { get; set; }
        public string Status { get; set; }
    }
}

File: hospital-management/src/app/shared/appointment.ts
export interface Appointment {
    id: number;
    patientId: number;
    doctorId: number;
    hospitalId: number;
    appointmentTime: string;
    appointmentType: string;
    isCancelled: boolean;
    cancellationReason: string;
    createdOn: string;
    createdBy: string;
}